website/content/docs/operators/deployment/schedulers/mesos-local-mac.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Setting up Heron with Mesos Cluster Locally on Mac
+---
+
+This is a step by step guide to run Heron on Mesos cluster locally.
+
+## Install Heron
```

Comment:
Change from "install heron" to "Get heron with Mesos Scheduler implementation" or someting like it. Otherwise this seems like a little duplicated with section "Compile Heron" below.

Or we maybe move this section to "Compile heron" and merge them together.